packages/mobile/src/components/bottom-tab-bar/BottomTabBar.tsx

```diff
@@ -104,10 +104,11 @@ export const BottomTabBar = (props: BottomTabBarProps) => {
     <Animated.View
       style={[styles.root, interpolatePostion(translationAnim, insets.bottom)]}
     >
-      <SafeAreaView
-        style={styles.bottomBar}
-        edges={['bottom']}
-        pointerEvents='auto'
+      <View
```

Comment:
Hmmm, we might want to test this + NowPlaying bar on android with navigation bar / navigation gestures enabled

Reply:
Will test on android more thoroughly

Reply:
needed some casing